go.mod

```diff
@@ -161,11 +165,12 @@ replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
 
 replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.3-patched
 
-// TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
-replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.6-cronos-revert-2
+replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.6-cronos-revert-3
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-15
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-16
 
 // Note: gorocksdb bindings for OptimisticTransactionDB are not merged upstream, so we use a fork
 // See https://github.com/tecbot/gorocksdb/pull/216
 replace github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.1.1
+
+replace github.com/tendermint/tendermint => github.com/crypto-org-chain/tendermint v0.34.20-0.20220630092134-c8c4a9fc4851
```

Comment:
https://github.com/tendermint/tendermint/compare/v0.34.19...crypto-org-chain:tendermint:fix-duplicated-tx-index

Reply:
does this bug also 0.6.x?

why not just patch tendermint v0.34.14 with the targeted commit (c8c4a9fc4851d4e49b1576c8071386b09ad1d2f5) ?

Reply:
yes, after the fix, the user doesn't need to patch the tx indexer after sync.

Reply:
> why not just patch tendermint v0.34.14 with the targeted commit

it's better to keep dependencies up to date?